config/locales/service_down.yml

```
@@ -0,0 +1,7 @@
+en:
+  service_down:
+    title: Sorry, there’s a problem with the service
+    try_later: Try again later.
+    progress_not_saved: If you reached this page after submitting information then it has not been saved. You’ll need to enter it again when the service is available.
+    questions_intro: If you have any questions, please email us at
```

For consistency, can we change this to match the service unavaiable page:
https://qa.apply-for-teacher-training.service.gov.uk/rails/view_components/service_unavailable_component/service_unavailable_page

```suggestion
    questions_intro: If you have any questions, email
```